OSDOCS-7640: Doc how service accounts assume AWS IAM roles (#64331)
openshift/openshift-docs: authentication/assuming-an-aws-iam-role-for-a-service-account.adoc

[role="_abstract"]
ifdef::openshift-rosa[]

> In {product-title} clusters that use the AWS Security Token Service (STS), the OpenShift API server can be enabled to project signed service account tokens which then can be used to assume an AWS Identity and Access Management (IAM) role in a pod. If the assumed IAM role has the required AWS permissions, the pods can authenticate against the AWS API using temporary STS credentials to perform AWS operations.

Reviewer: Suggestion: "...the OpenShift API server can be enabled to project signed service account tokens that can be used to assume an AWS..."

[role="_abstract"]
ifdef::openshift-rosa[]
In {product-title} clusters that use the AWS Security Token Service (STS), service accounts can be enabled to automatically assume an AWS Identity and Access Management (IAM) role in a pod. If the assumed IAM role has the required AWS permissions, the pods can run AWS SDK operations by using temporary STS credentials.

> The workflow by which a service account assumes an IAM role depends on whether the service account is hosted in a default project or in a user-defined project.

Reviewer 2: ```suggestion
```

endif::openshift-rosa[]

> You can use the pod identity webhook to enable a service account to automatically assume an AWS Identity and Access Management (IAM) role in your own pods. If the assumed IAM role has the required AWS permissions, the pods can run AWS SDK operations by using temporary STS credentials.

Reviewer 2: ```suggestion
You can use the pod identity webhook to project service account tokens to assume an AWS Identity and Access Management (IAM) role for your own workloads. If the assumed IAM role has the required AWS permissions, the pods can run AWS SDK operations by using temporary STS credentials.
```
